src/service/tables/previous_service_identity.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "ccf/crypto/pem.h"
+#include "ccf/kv/value.h"
+
+#include <string>
+#include <vector>
+
+namespace ccf
+{
+  using PreviousServiceIdentity = ServiceValue<crypto::Pem>;
+
+  namespace Tables
+  {
+    static constexpr auto PREVIOUS_SERVICE_IDENTITY =
```

Comment:
This new table should be documented in `builtin_maps.rst`. Then, I remembered we had a test to check that we document all tables but the CI seems happy. I guess that's because the table is only created on recovery?

Reply:
The happens at the end of the testsuite, and so should include a recovery?

Reply:
https://github.com/microsoft/CCF/blob/main/tests/suite/test_suite.py#L133

Reply:
Yes, but only in the Daily (which wasn't previusl run on this PR, but is now failing).